src/components/user/LoginForm/index.js

```diff
@@ -2,35 +2,35 @@ import React from 'react';
 import { bool, func, string } from 'prop-types';
 import { Field, reduxForm } from 'redux-form/immutable';
 import {
-  ActivityIndicator, Button, Text, View,
+  ActivityIndicator, Button, Text, View, TouchableOpacity
 } from 'react-native';
 
 import Input from 'components/common/Input';
 import * as constraints from 'utils/constraints';
 import translate from 'utils/i18n';
 import styles from './styles';
+import constants from 'styles/common';
 
 const LoginForm = ({ handleSubmit, error, submitting }) => (
-  <View onSubmit={handleSubmit}>
+  <View style={styles.container} onSubmit={handleSubmit}>
     {error && <Text>{error}</Text>}
     <Field
       name="email"
-      label={translate('SIGN_IN.email')}
+      label={translate('SIGN_IN.username').toUpperCase()}
       component={Input}
     />
     <Field
       name="password"
-      label={translate('SIGN_IN.password')}
+      label={translate('SIGN_IN.password').toUpperCase()}
       component={Input}
       password
     />
     { submitting ?
       <ActivityIndicator /> :
-      <View style={styles.button}>
-        <Button
-          title={translate('SIGN_IN.button')}
-          onPress={handleSubmit}
-        />
+      <View style={styles.submitButtonContainer}>
```

Comment:
You could put the activity indicator here and give this View a fixed height. This will avoid the layout changes you are having when the activity indicator replaces the submit button.